### PR TITLE
Update duckduckgo.po

### PR DESCRIPTION
**Pull request by ali-demirtas**
Turkish language updated.

### CODE REVIEW

File: locales/tr_TR/LC_MESSAGES/duckduckgo.po [hunk 1]
```diff
@@ -747,151 +750,152 @@ msgid "Clear All"
...
-msgstr "Eklenti Türleri altındaki %sArama Sağlayıcıları%s'na tıklayın"
+msgstr "Eklenti Türleri altındaki %sArama Sağlayıcıları%s öğesine tıklayın"
 
 msgid "Click on the %sTools icon%s in the top-right of the browser"
-msgstr "Tarayıcının sağ üst kısmındaki %sAraçlar simgesi%s'ne tıklayın"
+msgstr "Tarayıcının sağ üst tarafındaki %sAraçlar simgesine%s tıklayın"
 
 msgid "Click on the %sellipsis icon%s in the toolbar"
-msgstr "Araç çubuğundaki %sEksilti simgesi%s'ne tıklayın"
+msgstr "Araç çubuğundaki %üç nokta simgesine%s tıklayın"
```
Comment (jbarrett) [inline locales/tr_TR/LC_MESSAGES/duckduckgo.po]:
```suggestion
msgstr "Araç çubuğundaki %süç nokta simgesine%s tıklayın"
```

File: locales/tr_TR/LC_MESSAGES/duckduckgo.po [hunk 2]
```diff
@@ -747,151 +750,152 @@ msgid "Clear All"
...
+msgstr "DuckDuckGo eklentisini indirmek için %sburaya%s tıklayın"
 
 msgid ""
 "Click %sOK%s in the popup. %sNOTE%s: If OK button is disabled, it was "
 "already added. Click %sCancel%s and proceed to the next step."
 msgstr ""
-"Açılan pencerede %sTamam%s 'a tıklayın. %sNOT%s :Eğer Tamam butonu aktif "
-"değilse, hali hazırda eklenmiştir. %sİptal%s 'e basın ve bir sonraki adıma "
-"ilerleyin."
+"Açılır pencereden %sTamam%s seçeneğine tıklayın. %sNOT%s: %sTamam%s düğmesi "
```
Comment (jbarrett) [inline locales/tr_TR/LC_MESSAGES/duckduckgo.po]:
```suggestion
"Açılır pencereden %sTamam%s seçeneğine tıklayın. %sNOT%s: Tamam düğmesi "
```

**Comment by ali-demirtas** [comment]
Yes, the translations are correct

**Comment by jbarrett** [comment]
Thanks @ali-demirtas! Could you commit the suggestions? I don't have access to your fork.

**Comment by ali-demirtas** [comment]
@jbarrett I fixed the incorrect strings. :)

**Comment by jbarrett** [comment]
Thank you @ali-demirtas! :+1:

**Pull request merged**